### PR TITLE
improving tests in test_cells and test_pick

### PR DESCRIPTION
**Pull request by andrehora**
## Type of changes

- [ ] Bug fix
- [ ] New feature
- [ ] Documentation / docstrings
- [x] Tests
- [ ] Other

## Checklist

- [x] I've run the latest [black](https://github.com/psf/black) with default args on new code.
- [x] I've updated CHANGELOG.md and CONTRIBUTORS.md where appropriate.
- [x] I've added tests for new code.
- [x] I accept that @willmcgugan may be pedantic in the code review.

## Description

It seems that `test_cells` misses some inputs like boundary cases. Those are important to avoid regressions. I updated `test_cell_len_long_string` and created `test_cell_len_short_string` to overcome this possible issue.

Also improved `test_pick`to cover untested inputs of `pick_bool`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changed
+
+- Updated test_cells to cover boundary cases of cell_len
```
Comment (willmcgugan) [inline CHANGELOG.md:12]: No need for the changelog. A user of Rich doesn't care about tests.

**Review by willmcgugan — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (andrehora) [reply CHANGELOG.md]: Sure!

**Review by willmcgugan — APPROVED** [review]
